TrainingProject/TrainingProject/Controllers/HomeController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using TrainingProject.Models;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace TrainingProject.Controllers
+{
+    public class HomeController : Controller
+    {
+        string strconnect = string.Empty;
+
+        public HomeController()
+        {
+            strconnect = @"Data Source=172.20.21.129;MultipleActiveResultSets=True;Initial Catalog=RHPM;User ID=RHPM;Password=evry@123";
+        }
+
+        public int Product_ID { get; private set; }
+
+        // GET: Home
+        public ActionResult Products_Index()
```

Comment:
Move your code to new product controller; remove from home controller

Reply:
Done Sir